{{cookiecutter.project_slug}}/frontend/src/store/modules/main.ts

```diff
@@ -0,0 +1,252 @@
+import { api } from "@/api";
+import { getLocalToken, removeLocalToken, saveLocalToken } from "@/utils";
+import router from "@/router";
+import { AxiosError } from "axios";
+import { VuexModule, Module, Mutation, Action } from "vuex-module-decorators";
+import { IUserProfile, IUserProfileUpdate, IAppNotification } from "@/interfaces";
+import { UNAUTHORIZED } from "http-status-codes";
+
+@Module({ name: "main" })
+export default class MainModule extends VuexModule {
+  token = "";
+  isLoggedIn: boolean | null = null;
```

Comment:
I don't know why isLoggedIn should ever be null? Isn't is either false or true (false by default)?

Reply:
This is also something taken over from the main branch. `null` is the default state for `isLoggedIn` and you have an explicit `isLoggedIn===null`-check in `App.vue` which determines if the circular progress bar should be shown or not.
I think this is used to indicate the initial state (ie. the app is still trying to connect to the authentication backend).

Reply:
Ok, this seems correct then :)